src/lib/registration/ssl_certificate.rb

```diff
@@ -81,33 +78,17 @@ def self.download(url, insecure: false)
     # @return [Boolean] true if update was successful; false otherwise.
     #
     # @see CA_CERTS_DIR
-    # @see TMP_CA_CERTS_DIR
     def self.update_instsys_ca
-      # Update database
+      # Extract system certs in openssl and pem formats
       Yast::Execute.locally("trust", "extract", "--format=openssl-directory", "--filter=ca-anchors",
-        "--overwrite", TMP_CA_CERTS_DIR)
-
-      # Copy certificates/links
-      files = Dir[File.join(TMP_CA_CERTS_DIR, "*")]
-      return false if files.empty?
-      targets = ["pem", "openssl"].map { |d| File.join(CA_CERTS_DIR, d) }
-      new_files = targets.each_with_object([]) do |subdir, memo|
-        FileUtils.mkdir_p(subdir) unless Dir.exist?(subdir)
-        files.each do |file|
-          # FileUtils.cp does not seem to allow copying the links without dereferencing them.
-          Yast::Execute.locally("cp", "--no-dereference", "--preserve=links", file, subdir)
-          memo << File.join(subdir, File.basename(file))
-        end
-      end
-
-      # Cleanup
-      FileUtils.rm_rf(TMP_CA_CERTS_DIR)
+        "--overwrite", File.join(CA_CERTS_DIR, "openssl"))
```

Comment:
The goal of the code above is to keep the original content. The `/var/lib/ca-certificates` directory contains the initial trusted certificates. The `--overwrite` option causes removal of the previous content and losing the default SSL certificates.

That means we would have troubles accessing the other SSL secured sites.

Unfortunately after removing the `--overwrite` option the command fails with error `directory already exists`. So we need to extract the files into a temporary location and copy them to the correct target directory.

Reply:
OK. Then we need the copy code. I'll add it back.

Reply:
done